packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { AmountInput, TransferAccountSelector } from '@deriv/components';
+import { getDecimalPlaces } from '@deriv/shared';
+import { useStore, observer } from '@deriv/stores';
+import { localize, Localize } from '@deriv/translations';
+import { transfer_accounts } from './mock_accounts/mock_accounts';
+import type { TAccount } from './types';
+import './wallet-transfer.scss';
+
+const Divider = () => <div className='wallet-transfer__divider' />;
+
+const WalletTransfer = observer(() => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const [from_account, setFromAccount] = React.useState<TAccount>(transfer_accounts.wallets[0]);
+    const [to_account, setToAccount] = React.useState<TAccount>();
+    const [from_amount, setFromAmount] = React.useState();
+    const [to_amount, setToAmount] = React.useState();
+
+    const account_list_height_with_offset = 'calc(100vh - 12.2rem)';
+
+    const portal_id = is_mobile ? 'mobile_list_modal_root' : 'modal_root';
+
+    const to_account_list = React.useMemo(() => {
+        if (from_account?.label === 'Demo USD Wallet') {
+            setToAccount(undefined);
+            return { accounts: transfer_accounts.accounts, wallets: [] };
+        }
+        setToAccount(transfer_accounts.wallets[0]);
+        return { wallets: transfer_accounts.wallets, accounts: [] };
+    }, [from_account?.label]);
+
+    const transfer_to_hint = React.useMemo(() => {
+        return to_account?.label === 'Demo USD Wallet' ? (
```

Comment:
you need to check localize for label
```suggestion
        return to_account?.label === localize('Demo USD Wallet') ? (
```